### DISCUSSION TITLE
Milvus Community Forum will be the primary support platform!

### DISCUSSION THREAD

**Discussion by kateshaowanjou** [milvus-io/milvus#11111] [Announcement]
Hi everyone!
We are happy to announce that we are officially launching the [Milvus Community Forum](https://discuss.milvus.io/)!
While GitHub Discussions remains an available channel, moving forward **the Forum will be our primary platform for community support.** We strongly encourage every community member to give the Forum a shot and get answers more quickly!

We made this decision to provide a more organized platform as the Forum essentially functions as a repository of information for all community members. It is also a place where you can share your thoughts and work and have open, in-depth discussions with the rest of the community about anything related to Milvus. 

In an effort to build a more vibrant community, each month the most active members will be featured in a post as **Star Members of the Month** and awarded a digital badge that can be added to their LinkedIn profiles.
Come say "Hi" to everyone and start making some "noise"! We look forward to seeing you around! 

Kate Shao from Milvus Community Team

**Comment by solomonmanuelraj** [comment]
Hi Milvus team,  i need your help to clarify my doubts. i want to retrieve the source images based on Patch Query Image similarity. Patch Query image will contains some small /medium patch objects from the Source Images. what type of embedding & index types will be useful for this use case.  Expected input is Patch Query Image. Expected Output is Retrieving the Source Image which has the patch image.
with thanks
SM

**Reply by yhmo** [reply]
Okay, you activate this conversation which was created 2 years ago. Now we are mainly discussing in [Slack](https://slack.milvus.io/) and [Discussion panel](https://github.com/milvus-io/milvus/discussions).

The answer of your question:
You need to use an embedding tool to convert the patch objects into embeddings, then insert these embeddings into Milvus. The Milvus 2.x allows you input some scalar fields along with the vector field, so you can create a collection with a vector field and a varchar field that stores the id or path of the source image. When you query a patch image, you need an embedding tool convert the image into the same dimension embedding, and use this embedding to search in the collection. Milvus will return topk results for you, each result represent a patch object, And, from the result, you can get the id or path of the source image so that you know which source it comes from.

**Comment by tusg** [comment]
Hi everyone,
we have a use case where in we want to have different query nodes for different collections.
e.g.
collection1 - has 40 million records with 512 dimension vector
collection2 - has 3 million records with 52 dimension vector.
we want these two collections to be accessed by two different query nodes that have different memory configurations, is it possible ??
I am unable to find much detail regarding this in the documentation. Any suggestion or link that can help me here?

**Reply by yhmo** [reply]
40M 512dim ------ raw data size is 512*4*40=80GB, default segment size is 512MB, there are 160+ segments
3M 52dim ------ raw data size is 52*4*3=624MB, default segment size is 512MB, there are 2 segments
These segments were assigned to two query nodes. The balance strategy use the `queryNode.cacheSize` to calculate memory usage, the default value of a query node's memory size is 32GB which is defined in the milvus.yaml:
```
queryNode:
  cacheSize: 32 # GB, default 32 GB, `cacheSize` is the memory used for caching data for faster query. The `cacheSize` must be less than system memory size.
```

If you want to set different memory size for query nodes, you need to change this value in their milvus.yaml. Assume you set 64GB for the node_1 and 128GB for the node_2. The balance strategy could assign 50+ segments to the node_1, and assign 130+ segments to the node_2.
The balance strategy is transparent to users. That means the 2 segments of collection2 could be assigned to any nodes
